src/main/scala/com/gu/contentapi/services/S3.scala

```diff
@@ -9,17 +9,25 @@ import scala.util.{ Failure, Success, Try }
 
 object S3 extends Logging {
 
+  private val retryDelayMs = 100
```

Comment:
This isn't much of a pause, but maybe it's enough

Reply:
fair point. I guess I was being conservative and assuming we can see if it is enough in practice and if not, we can raise it.